### DISCUSSION TITLE
AttributeError: 'LidGroup' object has no attribute 'WaterBalance'

### DISCUSSION THREAD

**Discussion by KillDragon** [pyswmm/pyswmm#369] [Q&A] [answered]
When I use the official sample code, I get the following error message：
![image](https://user-images.githubusercontent.com/15921876/197736748-9eb18729-5eaa-4143-b70d-caed492b1dd6.png)

```
from pyswmm import Simulation, LidGroups

with Simulation("../Samples/LID_Model.inp") as sim:
    lid_on_sub = LidGroups(sim)["S1"]  
    i=0
    for lid in lid_on_sub:        
        print(i)
        i=i+1
        for step in sim:
           print(lid_on_sub.WaterBalance.inflow)
```

**Comment by bemcdonnell** [comment, marked as answer]
@KillDragon sorry. It looks like you didn’t edit the template for your discussion.  I updated my comment based on @dokzai. 

Try this:
```python
from pyswmm import Simulation, LidGroups

with Simulation("../Samples/LID_Model.inp") as sim:
    lid_on_sub = LidGroups(sim)["S1"]  
    for step in sim:
        i=0
        for lid in lid_on_sub:
            i+=1
            print(lid.water_balance.inflow)
```

**Reply by bemcdonnell** [reply]
@jennwuu, I see where this user found this example code. Can you check the tutorial to confirm the correct sample code for this? 

https://pyswmm.readthedocs.io/en/latest/tutorial/tutorial.html#lid-groups

**Reply by dokzai** [reply]
Hey @bemcdonnell 

I looked into the issue and going through the code, it looks like `WaterBalance` should be `water_balance` as seen in https://github.com/OpenWaterAnalytics/pyswmm/blob/master/pyswmm/lidgroups.py#L227. 

I discussed with @jennwuu and the example provided in the tutorial should be corrected to:

```python
>>> from pyswmm import Simulation, LidGroups
>>>
>>> with Simulation('/testmodel.inp') as sim:
...     lid_on_sub = LidGroups(sim)["subcatch_id"]
...     for lid in lid_on_sub:
...             print(lid)
...     print(lid_on_sub[0])
...     for step in sim:
...             print(lid_on_sub.water_balance.inflow)
...             print(lid_on_sub.water_balance.evaporation)
```

Similarly @KillDragon's code snippet should be corrected to:

```python
from pyswmm import Simulation, LidGroups

with Simulation("../Samples/LID_Model.inp") as sim:
    lid_on_sub = LidGroups(sim)["S1"]  
    for step in sim:
        i=0
        for lid in lid_on_sub:
            i+=1
            print(lid.water_balance.inflow)
```

Refer to this test section for other water balance functionalities: https://github.com/OpenWaterAnalytics/pyswmm/blob/master/pyswmm/tests/test_lid.py#L75-L128

**Reply by KillDragon** [reply]
thanks，you are right

**Reply by bemcdonnell** [reply]
@dokzai Would you have any interest in making a quick PR to fix the documentation?
